lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -83,9 +83,10 @@ module Cardano.Wallet.Primitive.Types
     , computeUtxoStatistics
     , log10
 
-    -- * BlockchainParameters
-    , GenesisBlockParameters (..)
-    , BlockchainParameters (..)
+    -- * GenesisParameters
```

Comment:
```suggestion
    -- * Blockchain Parameters
```

Reply:
I think that "Network Parameters" probably makes more sense here, as `NetworkParameters` is the type that sits at the root of the tree, so to speak. (`NetworkParameters` includes both `GenesisParameters` and `ProtocolParameters`.)

Reply:
Fixed in 39b079cb6319a4dcb8c2ffea0c1bd9232f100717.